src-electron/generator/generation-engine.js

```diff
@@ -446,70 +446,72 @@ async function generateAllTemplates(
     disableDeprecationWarnings: false,
   }
 ) {
-  return queryPackage
-    .getPackageByParent(genResult.db, genTemplateJsonPkg.id)
-    .then((packages) => {
-      let generationTemplates = []
-      let helperPromises = []
-      let partialPromises = []
-      let overridePath = null
-
-      // First extract overridePath if one exists, as we need to
-      // pass it to the generation.
-      packages.forEach((singlePkg) => {
-        if (singlePkg.type == dbEnum.packageType.genOverride) {
-          overridePath = singlePkg.path
-        }
-      })
+  let packages = await queryPackage.getPackageByParent(
+    genResult.db,
+    genTemplateJsonPkg.id
+  )
+  let generationTemplates = []
+  let helperPromises = []
+  let partialPromises = []
+  let overridePath = null
+
+  let hb = templateEngine.hbInstance()
```

Comment:
`2. Desingletonize the handlebars instance.`


Doesn't this make the handlebar instances into a singleton?